pkgs/development/tools/gojsontoyaml/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "gojsontoyaml";
+  version = "unstable-2020-06-02";
```

Comment:
Can you please update the commit message and PR title to this version? Also please squash commits together.

Reply:
TIL git squash.

I think I did it correctly.